vtex/toolbelt
# Fix deprecation check

#### What is the purpose of this pull request?
Fix deprecation check.

#### What problem is this solving?
Updating the version to a non deprecated one from a deprecated one wouldn't release the deprecation lock.

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/deprecation-check && \
yarn && yarn watch
```
Change the `package.json` version to `2.90.1`. Run two commands, it should be enough to trigger the deprecation lock.
Change the `package.json` version to `2.90.0`. Run commands freely, it shouldn't trigger the deprecation lock in any time.

#### Types of changes

#### Chores checklist
